pkgs/tools/inputmethods/fcitx5/fcitx5-unikey.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, stdenv, fetchFromGitHub, cmake, extra-cmake-modules, fcitx5, fcitx5-qt
+, ninja, gettext, qt5 }:
```

Comment:
`qt5` subpackages are already in scope, since you used `libsForQt5.callPackage`:
```suggestion
, ninja, gettext, wrapQtAppsHook
}:
```

Reply:
Good catch, thank you!